website/server/libs/payments/groupPayments.js

```diff
@@ -12,6 +14,9 @@ import { // eslint-disable-line import/no-cycle
   getUserInfo,
   sendTxn as txnEmail,
 } from '../email';
+import { paymentConstants } from './constants';
+// eslint-disable-line import/no-cycle
```

Comment:
does this actually do anything, on a line by itself?

Reply:
hmmm I dunno if it was a different line before - I'll check and fix it

Reply:
sorry seems I lost the time yesterday - pushed now